android/sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=zulip/zulip-mobile
+defaults.org=zulip
+defaults.project=zulip-mobile
+auth.token=getTokenFrom https://sentry.io/api/
```

Comment:
For now replaced it with SENTRY-API-TOKEN-HERE. We need to replace this with API before every release.